graph_model/model.py

```diff
@@ -0,0 +1,35 @@
+import graph_model.trie_model as trie_model
+import json
+class GraphModel:
+    def __init__(self):
+        self.trie = trie_model.Trie()
+
+    def train(self):
+        try:
+            data_file = open('./temp/data/pre_proceed_data.txt', 'r')
+        except:
+            trie_model.data_preprocess()
+            data_file = open('./temp/data/pre_proceed_data.txt', 'r')
+
+        content = data_file.read().splitlines()
+        data_file.close()
+        data_list = []
+        for line in content:
+            data_list.append(line)
+
+        self.trie.add_data(data_list)
+        return self
+
+    def load(self):
+        with open('./temp/model.json', 'rt') as f:
```

Comment:
It's a bit hardcoded approach. Please create config.py inside the model package with the dictionary containing all paths.